spec/rubocop/cop/rails/action_controller_flash_before_render_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::ActionControllerFlashBeforeRender, :config do
+  it 'registers an offense when using `flash` before `render`' do
+    expect_offense(<<~RUBY)
+      flash[:alert] = "msg"
+      ^^^^^ Use `flash.now` before `render`.
+      render :index
+    RUBY
+
+    expect_correction(<<~RUBY)
+      flash.now[:alert] = "msg"
+      render :index
+    RUBY
+  end
+
+  it 'does not register an offense when using `flash.now` before `render`' do
+    expect_no_offenses(<<~RUBY)
+      flash.now[:alert] = "msg"
+      render :index
+    RUBY
+  end
+
+  it 'does not register an offense when using `flash` before `redirect_to`' do
+    expect_no_offenses(<<~RUBY)
+      flash[:alert] = "msg"
+      redirect_to "https://www.rubygems.org"
+    RUBY
+  end
+end
```

Comment:
Can you add the following test and it passes?

```suggestion
  end

  it 'does not register an offense when using `flash` only' do
    expect_no_offenses(<<~RUBY)
      flash[:alert] = "msg"
    RUBY
  end
end
```

Reply:
Done here: https://github.com/rubocop/rubocop-rails/commit/da3f7ef7ff123953a07868b060c038db79643280#diff-c8d2739a4065d73a564379fef819c503bf4908167b9dfedaaaa799b60b882456R17-R21